spec/middleman-prismic/commands/prismic_spec.rb

```diff
@@ -71,6 +71,18 @@
         to have_received(:query).
           with(prismic_query)
     end
+
+    it "downloads via prismic ref" do
+      stub_options
+      response = double("response", group_by: [], total_pages: 1)
```

Comment:
do we need to stub out `group_by` and `total_pages` here? or can we get away without them, or with a `spy`?

Reply:
Doesn't appear we can. I think this is the issue with this class, is that it does too much. It has too many dependencies. I would love to refactor.